packages/framework/src/Model/Customer/User/Role/CustomerUserRoleGroup.php

```diff
@@ -51,8 +51,24 @@ class CustomerUserRoleGroup extends AbstractTranslatableEntity
     public function __construct(CustomerUserRoleGroupData $customerUserRoleGroupData)
     {
         $this->translations = new ArrayCollection();
-        $this->roles = $customerUserRoleGroupData->roles;
         $this->uuid = $customerUserRoleGroupData->uuid ?: Uuid::uuid4()->toString();
+        $this->setData($customerUserRoleGroupData);
+    }
```

Comment:
are the constants DEFAULT_ROLE_GROUP_NAME and USER_ROLE_GROUP_NAME in this file used anywhere?

Reply:
You are right. They are not used. Removed: https://github.com/shopsys/shopsys/pull/3323/commits/12967f3bac2c7511cd32343a460ff3c34b933f44